### PR TITLE
Converting FHIR measure to QICore

### PR DESCRIPTION
**Pull request by sb-prateekkeerthi**
## PR Template

Jira Ticket: [MAT-0000](https://jira.cms.gov/browse/MAT-0000)
(Optional) Related Tickets:

### Summary
Converting FHIR measure to QICore

### All Submissions
* [x] This PR has the JIRA linked.
* [x] Required tests are included
* [x] No extemporaneous files are included (i.e. Complied files or testing results)
* [ ] This PR is into the **correct branch**.
* [x] All Documentation as needed for this PR is Complete (or noted in a TODO or other Ticket)
* [ ] Any breaking changes or failing automation are noted by placing a comment on this PR.

### DevSecOps
If there is a question if this PR has a security or infrastructure impact, please contact the Security or DevOps engineer assigned to this project to discuss it further.

* [ ] This PR has NO significant security impact (i.e. Changing auth methods, Adding a new user type, Adding a required but vulnerable package)

### Reviewers
By Approving this PR you are attesting to the following:

*  Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
*  The tests appropriately test the new code, including edge cases
*  If you have any concerns they are brought up either to the developer assigned, security engineer, or leads

### CODE REVIEW

File: lambda-function/src/utils/measureConversionUtils.ts [hunk 1]
```diff
@@ -111,10 +110,22 @@ const getMeasureLibraryNameAndCql = (matMeasure: MatMeasure): { cqlLibraryName:
...
+  if (matchedCqlWithRegExpression) {
+    let replacedTextCql = textCql;
+    textCqlAfterReplacing = matchedCqlWithRegExpression.reduce((matchedCql) => {
+      replacedTextCql = replacedTextCql.replace(
+        matchedCql,
+        matchedCql.replace(/^using FHIR.*/gm, "using QICore version '4.1.0'"),
+      );
+      return replacedTextCql;
+    }, replacedTextCql);
+  }
```
Comment (adongare) [inline lambda-function/src/utils/measureConversionUtils.ts:124]: Considering using FHIR version '4.0.1' is going to be constant, would this work?
`textCql.replace("using FHIR version '4.0.1'", "using QICore version '4.1.0'")`

Comment (sb-prateekkeerthi) [reply lambda-function/src/utils/measureConversionUtils.ts:124]: Yeah it does work, but I wasn't sure if the version value is constant(in our case it is 4.0.1) so I have used regular expression to match it and then replace it

**Review by RohitKandimalla — APPROVED** [review]

**Review by adongare — APPROVED** [review]
